tests/markov/msm/test_oom.py

```diff
@@ -0,0 +1,574 @@
+# This file is part of PyEMMA.
+#
+# Copyright (c) 2015, 2014 Computational Molecular Biology Group, Freie Universitaet Berlin (GER)
```

Comment:
All this is outdated right? Things that no longer appear to be the case:
- this is part of pyemma
- it's 2014/2015
- we are CMB

Reply:
yeah there are a couple of these throughout the codebase. Id rather change this all together in a separate pr

Reply:
makes sense